Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -762,6 +785,21 @@ def countFailures(self):
 
         return count
 
+    def commandComplete(self, cmd):
```

Comment:
Do we need this method? (Especially that we are using run () method)

Reply:
Oh I already moved the code to `run()` as seen above, I forgot to delete this oops.